crates/ursa-telemetry/src/lib.rs

```diff
@@ -28,7 +28,7 @@ impl TelemetryConfig {
     pub fn new(name: &str) -> Self {
         Self {
             name: name.to_string(),
-            log_level: None,
+            log_level: "INFO".to_string(),
```

Comment:
this can then reverted back to None

Reply:
https://github.com/fleek-network/ursa/pull/223/commits/acc01e7fc57f79afcd4300e3172bb6b462b7fc1a